helm/terraform-bucket-provision/values.yaml

```
@@ -16,6 +16,7 @@ resources:
 
 terraform:
   namespace_apps: ~
+  workspace: default
```

We need to leave this empty if we want the default value to be properly populated

```suggestion
  workspace: ~
```